simplepath/utils.py

```diff
@@ -50,3 +51,11 @@ def deepvars(data_object):
         return object_dict
     else:
         return data_object
+
+
+def divide(a, b):
```

Comment:
this seems unnecessary. lets rather go with Python3 style division - https://docs.python.org/3.4/library/operator.html#mapping-operators-to-functions

> ```
Division	a / b	truediv(a, b)
Division	a // b	floordiv(a, b)
```

btw, same is supported in Py2 so that should not cause any compatibility issues